tests/Behat/FeatureContext.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\rdf_skos\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Drupal\Core\Database\Database;
+use Drupal\DrupalExtension\Context\ConfigContext;
+use Drupal\Tests\rdf_skos\Traits\SkosImportTrait;
+
+/**
+ * Defines step definitions that are generally useful in this project.
+ *
+ * We are extending ConfigContext to override the setConfig() method until
+ * issue https://github.com/jhedstrom/drupalextension/issues/498 is fixed.
+ *
+ * @todo Extend DrupalRawContext and gather the config context when the above
+ * issue is fixed.
+ */
+class FeatureContext extends ConfigContext {
+
+  use SkosImportTrait;
+
+  /**
+   * Imports the SKOS test graphs and configures it to be used.
+   *
+   * @beforeScenario @skos
+   */
+  public function importSkosContent(BeforeScenarioScope $beforeScenarioScope): void {
+    $sparql = Database::getConnection('default', 'sparql_default');
+    $base_url = $this->getMinkParameter('base_url');
+    $this->import($base_url, $sparql);
+    $graphs = $this->getTestGraphInfo($base_url);
+    $config = [];
+    foreach ($graphs as $name => $graph) {
+      $config['skos_concept_scheme'][] = [
+        'name' => $name,
+        'uri' => $graph['uri'],
+      ];
+
+      $config['skos_concept'][] = [
+        'name' => $name,
+        'uri' => $graph['uri'],
+      ];
+    }
+
+    $this->setConfig('rdf_skos.graphs', 'entity_types', $config);
+  }
+
+  /**
+   * Deletes the SKOS test graphs.
+   *
+   * @afterScenario @skos
+   */
+  public function deleteSkosContent(AfterScenarioScope $afterScenarioScope): void {
+    $sparql = Database::getConnection('default', 'sparql_default');
+    $base_url = $this->getMinkParameter('base_url');
+    $this->clear($base_url, $sparql);
+  }
+
+  /**
+   * {@inheritdoc}
+   *
+   * @todo Remove when https://github.com/jhedstrom/drupalextension/issues/498
```

Comment:
Why we are not implementing this patch by the normal way? 
Adding the patch file to the composer.json?